content/doc/book/blueocean/pipeline-view.adoc

```diff
@@ -0,0 +1,17 @@
+---
+layout: section
+title: Blue Ocean Pipeline View
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+:wip: true
+
+= Blue Ocean Pipeline View
```

Comment:
We'll need a new title here, "Pipeline View" doesn't really map to anything in Blue Ocean